setup.py

```diff
@@ -50,7 +50,7 @@
     name='amundsen-databuilder',
     version=__version__,
     description='Amundsen Data builder',
-    url='https://www.github.com/lyft/amundsendatabuilder',
+    url='https://www.github.com/amundsen-io/amundsendatabuilder',
     maintainer='Lyft',
     maintainer_email='amundsen-dev@lyft.com',
```

Comment:
tiny nit: are these still accurate? I think not but I do not know what they should be...

Reply:
I doubt they're correct; @markgrover is there a replacement?

Reply:
Yeah, good catch. They aren't correct, my recommendation would be:
`maintainer='Amundsen TSC'`
`maintainer_email='amundsen-tsc@lists.lfai.foundation'`

But would love a second opinion from @feng-tao before we change it.

Reply:
lgtm, thanks for raising the issue.

Reply:
Included, thanks y'all